test/libsolidity/smtCheckerTests/inheritance/constructor_hierarchy_base_calls_with_side_effects_8.sol

```diff
@@ -0,0 +1,22 @@
+pragma experimental SMTChecker;
+
+contract A {
+	uint public x;
+	constructor(uint) {}
```

Comment:
@mijovic in this case, before this commit, because the parameter is unnamed the argument `f()` was not evaluated, causing `x = 0` in the end which was wrong. Now it works.